packages/cli/src/rpc/validators.ts

```diff
@@ -224,10 +225,12 @@ export const validators = {
           message: `invalid argument ${index}: enr must be string`,
         }
       }
-      if (params[index].substr(0, 3) !== 'enr') {
+      if (isValidEnr(params[index])) {
         return {
           code: INVALID_PARAMS,
-          message: `invalid argument ${index}: string must start with 'enr:'`,
+          message: `invalid argument ${index}: string must be compliant with \
```

Comment:
This shouldn't be multiline as this is part of a JSON response object returned by the RPC.